src/content/blog/cross-assignments-in-the-navy-reserve.md

```diff
@@ -46,8 +46,7 @@ TRUIC
 - Travel Support
 - Reserve Pay
 - Billet Assignments
-- Supply 
-- Uniforms and Berthing
+- Supply - Uniforms and Berthing
```

Comment:
You need to fix your branch. This was something you already fixed and in this PR you are undoing it.